### PR TITLE
introduce impact and smaller fixes

### PR DESCRIPTION
**Pull request by atomic111**
Signed-off-by: Patrick Münch <patrick.muench1111@gmail.com>

### CODE REVIEW

File: core/mondoo-tls-security.mql.yaml [hunk 1]
```diff
@@ -55,83 +56,105 @@ policies:
...
+    impact: 95
     query: |
       tls.certificates.first {
+        subject.commonName
         notBefore < time.now
         notAfter - notBefore < 398*time.day
       }
   - uid: mondoo-tls-security-cert-no-cert-expired
-    title: None of the certificates have expired
+    title: Certificate has not expired
```
Comment (scottford-io) [inline core/mondoo-tls-security.mql.yaml:85]: Doe this one need an `impact`?

**Review by scottford-io — APPROVED** [review]
I think one control is missing an impact, but otherwise looks good!

**Review by chris-rock — APPROVED** [review]
Thank you @atomic111 Great improvement

File: core/mondoo-tls-security.mql.yaml (hunk 1 above)
Comment (chris-rock) [reply core/mondoo-tls-security.mql.yaml:85]: It does not require impact, since the impact is defined by the query.

**Pull request merged**